src/features/shared/sliding/SlidingItem.tsx

```diff
@@ -45,19 +47,54 @@ const OptionContainer = styled.div<{ active: boolean }>`
     `}
 `;
 
+const custom_slash_lengths: Dictionary<number> = {
+  [bookmarkOutline]: 35,
+  [mailUnread]: 40,
+};
+
+const SlashedIcon = styled(IonIcon)<{
+  icon: string;
+  slash: boolean;
+  bgColor: string;
+}>`
+  ${({ icon, slash, bgColor }) =>
+    slash &&
+    css`
+      &::after {
+        content: "";
+        position: absolute;
+        height: ${custom_slash_lengths[icon] ?? 30}px;
+        width: 3px;
+        background: var(--ion-color-${bgColor}-contrast);
```

Comment:
Let's always do white like before. (the bookmark should be solid white)

Reply:
The bookmark is black because the green is light enough to make black the better contrast; with white it looks kind of weird. I can make it white but it will reduce contrast in dark mode to a bad level (1.97:1 vs black is 10.65:1).

Reply:
That's OK, make it white, I am going to tweak the green color in a separate PR

Reply:
@rsammelson it still is black

Reply:
I guess I accidentally marked this as done with the others, I'll fix this